src/plasoscaffolder/dal/sqlite_query_execution.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=no-member
+# pylint does not recognize connect and close as member
+"""Base for sql query validators"""
+import sqlite3
+
+from plasoscaffolder.dal import base_sql_query_execution
+
+
+class SQLQueryExecution(base_sql_query_execution.BaseSQLQueryExecution):
+  """Class representing the SQLite query validator
+  """
+
+  def __init__(self, database_path: str):
+    """Initializes the SQL Query Validator
+
+    Args:
+      database_path: the path to the sqlite database schema
+    """
+    self._connection = sqlite3.connect(database_path)
```

Comment:
I just released, that i did not forget this. 
I check it when i initialize it. 
Do you want to have the exception handling in the sqlite query execution class at the connect.
Or is it fine like this (code snipped inside of sqlite_controller):
```
  def _IsDatabaseFile(self, path: str) -> bool:
    """Try to open the database File
    
    Args:
      path (str): the database file path
      
    Returns:
      bool: if the file can be opened and is a database file"""
    try:
```
      execution = sqlite_query_execution.SQLQueryExecution(path)
    except sqlite3.OperationalError:
      return False
    self._query_execution = execution
    return True